weather_forecast_retrieval/data/hrrr/file_handler.py

```diff
@@ -13,6 +15,7 @@ class FileHandler:
 
     FOLDER_NAME_BASE = 'hrrr.{}'
     FILE_NAME_BASE = 'hrrr.t{:02d}z.wrfsfcf{:02d}.{}'
+    FILE_PATTERN = r'hrrr.t(\d\d)z.wrfsfcf(\d\d).grib2'
```

Comment:
Couldn't this be just 
```python
FILE_NAME_BASE + '.grib2'
```

Would be nice to only maintain one regex for the file name

Reply:
They are two different things, one is for formatting a file name and the other is a regex pattern. So not sure how it would look to maintain only one since the formats for format/regex are different.

Reply:
Got it.
I see this as a potential source for error, where the HRRR file name changes and you forget to change both. Would have to think about a solution how both would be possible to be maintained at the same time.

Reply:
It is a potential source but should be caught by the tests. One possible way is to use `FILE_NAME_BASE` and somehow create the regex string with `.format()`.